mne/realtime/epochs.py

```diff
@@ -314,6 +314,7 @@ def _process_raw_buffer(self, raw_buffer):
         verbose = 'ERROR'
         sfreq = self.info['sfreq']
         n_samp = len(self._raw_times)
+        prev_samp = int(self._find_events_kwargs['min_samples'] - 1)
```

Comment:
you need to check that this is not negative ...

and also deal with the case when it's not provided -- in which case you use the default value